src/Components/Testimonials/TestimonialsForm.js

```diff
@@ -1,32 +1,96 @@
-import React, { useState } from 'react';
-import '../FormStyles.css';
+import {useEffect, useState} from 'react';
+import {Formik, Form, Field} from 'formik';
+import { ErrorsForm } from '../common/messagesForm';
+import { CKEditor } from '@ckeditor/ckeditor5-react';
+import { UrlInput } from '../common/getUrlInputFile';
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+import { validateFormTestimonials, replaceCKeditor } from '../common/validateFormTestimonials';
+import {getTestimonial,createTestimonial,modifiedTestimonial} from '../../Services/testimonialService';
+const TestimonialForm = ({testimonialId}) => {
 
-const TestimonialForm = () => {
-    const [initialValues, setInitialValues] = useState({
-       name: '',
-       description: '' 
-    });
+  const [imgState,setImgState] = useState();
+  const [dataForm, setDataForm] = useState({
+    name:'',
+    description:'',
+    image: ''
+  });
 
-    const handleChange = (e) => {
-        if(e.target.name === 'name'){
-            setInitialValues({...initialValues, name: e.target.value})
-        } if(e.target.name === 'description'){
-            setInitialValues({...initialValues, description: e.target.value})
-        }
-    }
+  useEffect(() => {
+    if(testimonialId) setDataForm(getTestimonial(testimonialId));
+  }, [testimonialId]);
 
-    const handleSubmit = (e) => {
-        e.preventDefault();
-        console.log(initialValues);
-    }
+  const handleChaneCkEditor = (editor, setFieldValue) => {
```

Comment:
handleChangeDescription maybe its a better name. Its using CkEditor but its changing the description and that's the main functionallity of this function.